sdk/storage/src/common/crypt.cpp

```diff
@@ -150,6 +214,16 @@ namespace Azure { namespace Storage {
 
 #else
 
+  std::string Sha256(const std::string& text)
```

Comment:
I strongly recommend self-written hashing/encryption algorithms, although leveraging other dependencies, to implement some predefined input/output UT for validation purposes. Then if a maintainer changes any code that may have an impact on the output, it will raise more alarms.

Reply:
what do you mean `self-written hashing/encryption algorithms`? can you clarify?

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/430